templates/05-group-validation-webhook.service.yaml.tmpl

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
We're having a single service for all the webhooks, right? If so, it seems to me that we shouldn't have `group-validation` in the file name

Reply:
Good call.